lib/src/utils.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:math';
+
+import 'package:vector_math/vector_math_64.dart';
+
+class Utils {
```

Comment:
We are trying to avoid generic utility classes since it's often viewed as an anti-pattern. Maybe it could just be named something more specific?

Reply:
We don't have this rule active, but it's related: https://dart-lang.github.io/linter/lints/avoid_classes_with_only_static_members.html

Reply:
ok, i made them simple global variables